community/values/src/main/java/org/neo4j/values/storable/LocalDateTimeValue.java

```diff
@@ -317,13 +318,13 @@ ZoneId getZoneId( Supplier<ZoneId> defaultZone )
     @Override
     ZoneId getZoneId()
     {
-        throw new UnsupportedTemporalTypeException( "Cannot get the timezone of" + this );
+        throw new UnsupportedTemporalUnitException( "Cannot get the timezone of: %s" + this );
```

Comment:
Don't you need a String.format with the `%s` ?

Reply:
Yes, I do. And I have missed it in more places...